adapters/src/chainlink.e2e.ts

```diff
@@ -0,0 +1,96 @@
+import nock from 'nock'
```

Comment:
Let's move the tests to `__tests__` and call it `chainlink.test.ts` from now on?

Reply:
yea sure